contracts/HasherMock.sol

```diff
@@ -0,0 +1,44 @@
+pragma solidity ^0.4.23;
+
+// Copyright 2017 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+// Common: HasherMock
+//
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+
+import "./Hasher.sol";
+
+/// @title HasherMock
+/// @dev Implements a keccak256 hashing function for OpenSTValueMock.sol
+contract HasherMock is Hasher {
+
+	/**
```

Comment:
this is section comment like below
/* Public pure functions */

Reply:
Done!